cf_xarray/accessor.py

```diff
@@ -835,7 +830,7 @@ def __getattr__(self, attr):
             obj=self._obj.plot,
             attr=attr,
             accessor=self.accessor,
-            key_mappers=dict.fromkeys(self._keys, (_single(_get_all),)),
+            key_mappers=dict.fromkeys(self._keys, (_single(_get_coords),)),
```

Comment:
What about `scatter`? Should we add an if statement?
Also, just `get_coords`, right? For `cf_xarray` dims and indexes should be coords as well?

Reply:
Yes, looks like `_get_all` is OK.